tokens/baked.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "0xa4cb0dce4849bdcad2d553e9e68644cf40e26cce",
+    "aurora_address": "0xa4cb0dce4849bdcad2d553e9e68644cf40e26cce",
+    "name": "BakedToken",
+    "symbol": "BAKED",
+    "decimals": 18,
+    "icon": "http://res.cloudinary.com/dswdlvq6b/image/upload/v1634828024/hregigyheev1rgmbwi86.svg",
```

Comment:
Please check the README, it should be a data URL, not URL. Either follow the following steps and fill it with the optimized SVG in the data URL or leave it empty (I can't commit to your repo to change it).